Dockerfile

```diff
@@ -10,4 +10,6 @@ USER app
 
 COPY . /usr/src/app
 
+ENV NODE_PATH /code/node_modules
```

Comment:
Nope - by default it is nothing and is only used when the module can't be found elsewhere:

> If the NODE_PATH environment variable is set to a colon-delimited list of absolute paths, then node will search those paths for modules if they are not found elsewhere.

*- https://nodejs.org/api/modules.html#modules_loading_from_the_global_folders*